Sources/Masse/PodcastEntry.swift

```diff
@@ -16,12 +16,30 @@ struct PodcastEntry {
     let date: Date
     let filename: String
     
-    init(meta: [String: String], filename: String) {
+    init(meta: [String: String], filename: String, folder: String) {
         self.meta = meta
         self.filename = filename
         let dateString = meta[Keys.PodcastEntry.date.rawValue].expect("Need \(Keys.PodcastEntry.date.rawValue) entry in podcast entry \(filename)")
         date = PodcastEntry.dateFormatter.date(from: dateString).expect("Invalid formatted date \(dateString) in post \(filename)")
+        
         // insert the podcast-format date into the meta
         self.meta["podcastDate"] = PodcastEntry.podcastDateFormatter.string(from: date)
+        
+        // calculate the duration
+        if let mp3Filename = meta[Keys.PodcastEntry.file.rawValue] {
```

Comment:
Let's convert this (and the next `if let`) to `guard let`s if that doesn't hinder the reading too much, to prevent some of this pyramid-of-doom.